### ISSUE TITLE
Error during build

### ISSUE THREAD

**Issue by KoljaL** [basuke/vite-plugin-sveltekit-php-backend#2]
Running in `dev mode` works fine, but if I try to build the app, I receive this: 

```console
vite v4.0.0 building SSR bundle for production...
✓ 6 modules transformed.
[vite-plugin-sveltekit-php-backend] ENOENT: no such file or directory, open '/Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/.svelte-kit/output/server/php/src_routes_pherdle_+page.server.php'
file: /Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/src/routes/pherdle/+page.server.php
error during build:
Error: ENOENT: no such file or directory, open '/Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/.svelte-kit/output/server/php/src_routes_pherdle_+page.server.php'
    at Object.openSync (node:fs:601:3)
    at Object.writeFileSync (node:fs:2249:35)
    at Object.transform (file:///Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/node_modules/.pnpm/vite-plugin-sveltekit-php-backend@0.5.0/node_modules/vite-plugin-sveltekit-php-backend/dist/index.js:65:20)
    at Object.handler (file:///Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/node_modules/.pnpm/vite@4.0.0/node_modules/vite/dist/node/chunks/dep-ed9cb113.js:45823:19)
    at file:///Users/kolja/Code/vite-plugin-sveltekit-php-backend2/vite-plugin-sveltekit-php-backend/example/node_modules/.pnpm/rollup@3.23.0/node_modules/rollup/dist/es/shared/node-entry.js:24656:40
 ELIFECYCLE  Command failed with exit code 1
```

System: MacOS, Node18, installed with `pnpm`

**Comment by basuke** [comment]
Thanks for trying! Yes, that's known issue. I'll figure out correct location and timing to create intermediate PHP files.

**Comment by basuke** [comment]
With 0.6.0 now you can build the app. Please try by yourself. Note `adapter-node` requires extra settings for form actions. https://github.com/sveltejs/kit/issues/10148

**Comment by KoljaL** [comment]
Hi @basuke ,
I have only just found the time to test version 0.6.
Unfortunately, I still get the same error.

I'm also not sure how I managed to output a server.php in dev mode the last time.

I use the build-in server of PHP (`php -S localhost:9000`) and have tried all (sensible) folders as root:  
 `./`, `./src/routes/`,`./src/`, 
Unfortunately, even then I only get error messages, like:   
`reject(new Error('Timeout: cannot receive value record in ${timeout} ms'));`  
or  
`throw er; // Unhandled 'error' event`

P.S.
Sorry for the PR, it was just a gimmick and actually i wanted to add it to my fork only.
